testsuite/tests/basic-more/opaque_prim.ml

```diff
@@ -0,0 +1,8 @@
+
+let f x = opaque x
+
+let () =
+  assert(f f == f);
+  assert(opaque 1 = 1);
+  assert(opaque 1. = 1.)
+
```

Comment:
I guess looking at the emitted IR is far too brittle ? It's to be expected, but I feel like this could break quite easily/expectantly.